### PR TITLE
Misuv 2387 1

### PR DESCRIPTION
**Pull request by jack-malik**
Implementation of changes to WhatYouOwe page incorporating frontend-hmrc.

### CODE REVIEW

File: test/views/WhatYouOweViewSpec.scala [hunk 1]
```diff
@@ -61,7 +61,7 @@ class WhatYouOweViewSpec extends TestSupport with FeatureSwitching with Implicit
       val anchor: Element = pageDocument.getElementById("sa-note-migrated").selectFirst("a")
       anchor.text shouldBe whatYouOwe.saLink
       anchor.attr("href") shouldBe "http://localhost:8930/self-assessment/ind/1234567890/account"
-      anchor.attr("target") shouldBe "_blank"
+//      anchor.attr("target") shouldBe "_blank"
```
Comment (chandraAndem) [inline test/views/WhatYouOweViewSpec.scala:64]: This test should not be removed.

File: test/views/WhatYouOweViewSpec.scala [hunk 2]
```diff
@@ -325,7 +324,7 @@ class WhatYouOweViewSpec extends TestSupport with FeatureSwitching with Implicit
 
         pageDocument.getElementById("payment-under-review-info").text shouldBe whatYouOwe.paymentUnderReviewPara
         paymentUnderReviewParaLink.attr("href") shouldBe "https://www.gov.uk/tax-appeals"
-        paymentUnderReviewParaLink.attr("target") shouldBe "_blank"
+//        paymentUnderReviewParaLink.attr("target") shouldBe "_blank"
```
Comment (chandraAndem) [inline test/views/WhatYouOweViewSpec.scala:327]: Same as above this test should not be removed as it tests openining in new tab.

File: test/views/WhatYouOweViewSpec.scala [hunk 3]
```diff
@@ -513,7 +512,7 @@ class WhatYouOweViewSpec extends TestSupport with FeatureSwitching with Implicit
...
```
Comment (chandraAndem) [inline test/views/WhatYouOweViewSpec.scala:515]: Same as above.

**Review by chandraAndem — CHANGES_REQUESTED** [review]
Opening in new tab functionality is missing

File: app/views/WhatYouOwe.scala.html [hunk 4]
```diff
@@ -17,43 +17,55 @@
...
+    @if(utr.isDefined) {
+        <div id="payments-due-note" class="govuk-inset-text">
 
-		    <p id ="sa-note-migrated">@messages("whatYouOwe.sa-note")
-			    <a href="@{appConfig.saViewLandPService(utr.get)}" target="_blank">@messages("whatYouOwe.sa-link")</a>.
-		    </p>
+            @p(content = Html(
+                messages("whatYouOwe.sa-note") +
+                        link(link = appConfig.saViewLandPService(utr.get), messageKey = "whatYouOwe.sa-link",
+                            outerMessage = Some(".")).toString()), id = Some("sa-note-migrated"))
```
Comment (david-walters-HMRC) [inline app/views/WhatYouOwe.scala.html:50]: the full stop shouldn't need to String

Comment (david-walters-HMRC) [inline app/views/WhatYouOwe.scala.html:50]: an id is also not needed

File: app/views/WhatYouOwe.scala.html [hunk 5]
```diff
@@ -63,121 +75,135 @@
...
+                <div class="govuk-grid-row">
                     @if(chargesList.balanceDetails.overDueAmount > 0) {
-                        <div id="overdueAmount" class="column-one-third">
-                            <p class="what-you-owe-total-heading">@messages("whatYouOwe.overdueAmount")</p>
-                            <p class="what-you-owe-total-values what-you-owe-total-values-bold">@chargesList.balanceDetails.overDueAmount.toCurrency</p>
+                        <div id="overdueAmount" class="govuk-grid-column-one-third">
+                            @p{
+                                @messages("whatYouOwe.overdueAmount")
+                            }
+                            <p class="govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold">@chargesList.balanceDetails.overDueAmount.toCurrency</p>
```
Comment (david-walters-HMRC) [inline app/views/WhatYouOwe.scala.html:190]: p component should be being used

File: app/views/WhatYouOwe.scala.html [hunk 6]
```diff
@@ -63,121 +75,135 @@
...
                     }
                     @if(chargesList.balanceDetails.balanceDueWithin30Days > 0) {
-                        <div id="balanceDueWithin30Days" class="column-one-third">
-                            <p class="what-you-owe-total-heading">@messages("whatYouOwe.balanceDueWithin30Days")</p>
-                            <p class="what-you-owe-total-values what-you-owe-total-values-bold">@chargesList.balanceDetails.balanceDueWithin30Days.toCurrency</p>
+                        <div id="balanceDueWithin30Days" class="govuk-grid-column-one-third">
+                            @p{
+                                @messages("whatYouOwe.balanceDueWithin30Days")
+                            }
+                            <p class="govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold">@chargesList.balanceDetails.balanceDueWithin30Days.toCurrency</p>
```
Comment (david-walters-HMRC) [inline app/views/WhatYouOwe.scala.html:198]: p component should be being used

File: app/views/WhatYouOwe.scala.html [hunk 7]
```diff
@@ -63,121 +75,135 @@
...
                     }
                     @if(chargesList.balanceDetails.totalBalance > 0) {
-                        <div id="totalBalance" class="column-one-third">
-                            <p class="what-you-owe-total-heading">@messages("whatYouOwe.totalBalance")</p>
-                            <p class="what-you-owe-total-values what-you-owe-total-values-bold">@chargesList.balanceDetails.totalBalance.toCurrency</p>
+                        <div id="totalBalance" class="govuk-grid-column-one-third">
+                            @p{
+                                @messages("whatYouOwe.totalBalance")
+                            }
+                            <p class="govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold">@chargesList.balanceDetails.totalBalance.toCurrency</p>
```
Comment (david-walters-HMRC) [inline app/views/WhatYouOwe.scala.html:206]: p component should be being used

File: app/views/WhatYouOwe.scala.html [hunk 8]
```diff
@@ -188,33 +214,32 @@ <h1 class="heading-xlarge" id="page-heading">@messages("whatYouOwe.heading")</h1
         <p>
```
Comment (david-walters-HMRC) [inline app/views/WhatYouOwe.scala.html:214]: p component should be being used

File: app/views/WhatYouOwe.scala.html [hunk 9]
```diff
@@ -188,33 +214,32 @@ <h1 class="heading-xlarge" id="page-heading">@messages("whatYouOwe.heading")</h1
...
             @paymentTypesDropdown
         </p>
-
         <p>
```
Comment (david-walters-HMRC) [inline app/views/WhatYouOwe.scala.html:217]: p component should be being used

File: app/views/WhatYouOwe.scala.html [hunk 10]
```diff
@@ -309,8 +338,9 @@ <h3 id="future-payments-heading" class="heading-large">@messages("whatYouOwe.fut
             <div>
                 @paymentDaysNoteParagraph
                 @creditOnAccountNotePanel
+
                 <div id="payment-button">
-                    <a id="payment-button-link" class="button"  role="button" href="@controllers.routes.PaymentController.paymentHandoff(chargesList.getEarliestTaxYearAndAmountByDueDate._2.toPence)"
+                    <a id="payment-button-link" class="govuk-button"  role="button" href="@controllers.routes.PaymentController.paymentHandoff(chargesList.getEarliestTaxYearAndAmountByDueDate._2.toPence)"
```
Comment (david-walters-HMRC) [inline app/views/WhatYouOwe.scala.html:343]: should be able to use link component here

**Pull request closed without merging**